how-to-docs.html

```diff
@@ -8,75 +8,87 @@
+generates new HTML files and commits them to the <code>gh-pages</code> branch.
+The content of the <code>gh-pages</code> branch is automatically deployed
+to <a href="https://cplusplus.github.io/LWG/">https://cplusplus.github.io/LWG/</a>.
+</p>
 
 <h2>Prerequisites</h2>
 <ul>
   <li>Git</li>
   <li>A C++17 compiler. Clang 9 works. GCC 8.x works for unix-like systems, but GCC 9.3 is recommended (because its <code>std::filesystem</code> supports Windows, and no additional <code>-lstdc++fs</code> linker option is needed).</li>
   <li>Windows is a prerequisite to use the Windows <code>.bat</code> scripts, or
-  a POSIX environment is a prerequisite to use the equivalent <code>.sh</code>
-  scripts.</li>
+  a POSIX environment is a prerequisite to use the equivalent <code>make</code>
+  commands.</li>
 </ul>
 
 <h2>Initial setup</h2>
 <p><b>Do this once only:</b></p>
 <blockquote>
 <pre>cd path-to-where-you-want-to-keep-this-stuff
 git clone git@github.com:cplusplus/LWG.git issues
-git clone -b gh-pages git@github.com:cplusplus/LWG.git issues-gh-pages
-cd issues
-mkdir mailing</pre>
+</pre>
 </blockquote>
-<p>Using recent versions of Git the second <code>git clone</code> command
-can be replaced by adding a new "worktree" that shares the repository metadata
-with the initial clone:</p>
-<blockquote><pre>
-git -C issues worktree add -b gh-pages ../issues-gh-pages origin/gh-pages
-</pre></blockquote>
-<p>This has the advantage of only cloning the repo once,
-which saves time, bandwidth and disk space.</p>
-
+<p>
+The commands shown below assume you are in the <code>issues</code> directory
+that contains the git repository cloned by the command above.
+</p>
+<p>
+The repository contains a <code>Makefile</code> that has recipes for
+building the software (<code>make pgms</code>),
+updating the meta-data (<code>make new-papers</code> etc.),
+and regenerating the HTML lists (<code>make lists</code>).
+</p>
 
-<h2>Build issues software</h2>
-<p>Do this after the initial setup, and then again whenever the source 
-code (in <code>issues/src</code>) changes.</p>
+<h2>Working with the issues lists on Windows</h2>
+<p>
+If you can't use <code>make</code> then the steps are a little more manual:
+</p>
 <blockquote>
-  <pre>cd issues
-bin\build_pgms</pre>
-</blockquote>
+<pre>cd issues
+mkdir mailing
+bin\build_pgms
+bin\build_lists
+</pre></blockquote>
+<p>
+You will need to re-run <code>bin\build_pgms</code>
+whenever the source code (in <code>issues/src</code>) changes.
+(This isn't needed when using <code>make</code> because the makefile
+will automatically do <code>make pgms</code> as a prerequisite of
+<code>make lists</code>.)
+</p>
 
 <h2>Add or change issues</h2>
 <p>There is one source file per issue, in <code>issues/xml</code>. The file
-<code>issues/xml/lwg-template.xml</code> can be used as a template to start a 
+<code>issues/xml/lwg-template.xml</code> can be used as a template to start a
 new issue. Use any text editor to edit the files.</p>
```

Comment:
I sincerely hope anyone who makes it here knows how to edit text files.
```suggestion
new issue.</p>
```